src/components/ProjectsPage/Projects.js

```diff
@@ -68,18 +63,100 @@ const Projects = ({
   const [selectedProjectsState, setSelectedProjectsState] = useState('active')
   const [sortProjectId, setSortProjectId] = useState('byName')
   const [deletingProjects, setDeletingProjects] = useState({})
+  const [jobsFilter, setJobsFilter] = useState({
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    }
+  })
+  const [workflowsFilter, setWorkflowsFilter] = useState({
+    groupBy: GROUP_BY_WORKFLOW,
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    },
+    state: STATE_FILTER_ALL_ITEMS
+  })
+  const [scheduledFilter, setScheduledFilter] = useState({
+    dates: {
+      value: [new Date(), new Date(moment().add(1, 'days'))]
+    }
+  })
+  const [loadingState, setLoadingState] = useState({
+    jobs: true,
+    workflows: true,
+    scheduled: true
+  })
+
   const abortControllerRef = useRef(new AbortController())
   const terminatePollRef = useRef(null)
-  const urlParams = useParams()
+  const navigate = useNavigate()
   const dispatch = useDispatch()
   const { isDemoMode } = useMode()
   const { isNuclioModeDisabled } = useNuclioMode()
-
+  const projectStore = useSelector(store => store.projectStore)
+  const { jobs, scheduled } = useSelector(store => store.jobsStore)
+  const { data: workflows } = useSelector(store => store.workflowsStore.workflows)
   const tasksStore = useSelector(store => store.tasksStore)
 
+  //
+  useEffect(() => {
+    dispatch(jobsActions.fetchJobs('*', jobsFilter)).then(() => {
+      setLoadingState(state => ({
+        ...state,
+        jobs: false
+      }))
+    })
+  }, [dispatch, jobsFilter])
+  //
+
+  //
+  useEffect(() => {
+    dispatch(workflowActions.fetchWorkflows('*', workflowsFilter)).then(() => {
+      setLoadingState(state => ({
+        ...state,
+        workflows: false
+      }))
+    })
+  }, [dispatch, workflowsFilter])
+  //
+
+  //
+  useEffect(() => {
+    dispatch(jobsActions.fetchScheduledJobs('*')).then(() => {
+      setLoadingState(state => ({
+        ...state,
+        scheduled: false
+      }))
+    })
+  }, [dispatch])
+  //
+
```

Comment:
Move all logic that are related to the new Monitoring template to a separate component (`ProjectsMonitoring`)
Also, I believe that we need to send a new request on selecting the filter for Scheduled jobs.